getting-started/deployment_guide.md

```diff
@@ -38,45 +30,234 @@ KubeArmor Host Policy Manager Deployment ...
+
+Lets apply the policy to block such execution:
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: block-pkg-mgmt-tools-exec
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  process:
+    matchPaths:
+    - path: /usr/bin/apt
+    - path: /usr/bin/apt-get
+  action:
+    Block
+EOF
+```
 
-### 3. Deploying sample app and policies
-   
-#### a. Deploy sample [multiubuntu app](../examples/multiubuntu.md)
+Now execute the `apt` command to download the `masscan` tool.
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/examples/multiubuntu/multiubuntu-deployment.yaml
+kubectl exec -it $POD -- sh -c "apt update && apt install masscan"
 ```
 
-#### b. Deploy [sample policies](security_policy_examples.md)
+It will be denied permission to execute.
+
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/examples/multiubuntu/security-policies/ksp-group-1-proc-path-block.yaml
+sh: 1: apt: Permission denied
+command terminated with exit code 126
 ```
-This sample policy blocks execution of `sleep` command in ubuntu-1 pods.
 
-#### c. Simulate policy violation
+</details>
+
+<details>
+  <summary><h4>Get policy violations notifications</h4></summary>
+
 ```
-$ POD_NAME=$(kubectl get pods -n multiubuntu -l "group=group-1,container=ubuntu-1" -o jsonpath='{.items[0].metadata.name}') && kubectl -n multiubuntu exec -it $POD_NAME -- bash
-# sleep 1
-(Permission Denied)
+karmor logs -n default --json
 ```
-### 4. Getting Alerts/Telemetry from KubeArmor
 
-#### Observing logs using karmor cli
+```json
+{
+  "Timestamp": 1686475183,
+  "UpdatedTime": "2023-06-11T09:19:43.451704Z",
+  "ClusterName": "default",
+  "HostName": "ip-172-31-24-142",
+  "NamespaceName": "default",
+  "PodName": "nginx-8f458dc5b-fl42t",
+  "Labels": "app=nginx",
+  "ContainerID": "8762eafc25a35ab90089f79703b86659989e8e547c2c029fb60f55d884355000",
+  "ContainerName": "nginx",
+  "ContainerImage": "docker.io/library/nginx:latest@sha256:af296b188c7b7df99ba960ca614439c99cb7cf252ed7bbc23e90cfda59092305",
+  "HostPPID": 3341922,
+  "HostPID": 3341928,
+  "PPID": 786,
+  "PID": 792,
+  "ParentProcessName": "/bin/dash",
+  "ProcessName": "/usr/bin/apt",
+  "PolicyName": "block-pkg-mgmt-tools-exec",
+  "Severity": "1",
+  "Type": "MatchedPolicy",
+  "Source": "/bin/dash",
+  "Operation": "Process",
+  "Resource": "/usr/bin/apt update",
+  "Data": "syscall=SYS_EXECVE",
+  "Enforcer": "BPFLSM",
+  "Action": "Block",
+  "Result": "Permission denied"
+}
 ```
-karmor logs
+
+</details>
+
+<details>
+  <summary><h4>Deny access to service account token</h4></summary>
+
+K8s mounts the service account token by default in each pod even if there is no app using it. Attackers use these service account tokens to do lateral movements.
+
+For e.g., to access service account token:
+```
+❯ kubectl exec -it $POD -- sh -c "cat /run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
Should we try executing something which tries to leverage the service account token, may show how the token is used/exploited? 

Like
```
curl https://$KUBERNETES_PORT_443_TCP_ADDR/api --insecure --header "Authorization: Bearer $(cat /run/secrets/kubernetes.io/serviceaccount/token)"
```

Reply:
Great point. Fixed.